src/app/modal/components/lose-changes/lose-changes.component.scss

```diff
@@ -0,0 +1,29 @@
+@import "node_modules/govuk-frontend/govuk/all";
+
+.mat-dialog-content {
+  overflow: hidden;
+}
+
+.title {
+  color: #0B0C0C;
+  font-size: 36px;
```

Comment:
can this work if we use these classes? https://design-system.service.gov.uk/styles/typography/